packages/language/src/preprocessor/compiler-options/translator.ts

```diff
@@ -1353,21 +1353,26 @@ translator.flag("goff", ["GOFF"], ["NOGOFF"]);
 translator.rule(["INCAFTER"], (option, options) => {
   ensureArguments(option, 1, 1);
   const value = option.values[0];
-  ensureType(value, "option");
-  if (value.name.toUpperCase() !== "PROCESS") {
-    throw new TranslationError(
-      value.token,
-      `Expected "PROCESS", but received '${value.name}'.`,
-      1,
-    );
+  if (value.kind === SyntaxKind.CompilerOption) {
+    ensureType(value, "option");
+    if (value.name.toUpperCase() !== "PROCESS") {
+      throw new TranslationError(
+        value.token,
+        `Expected "PROCESS", but received '${value.name}'.`,
+        1,
+      );
+    }
+    ensureArguments(value, 1, 1);
+    const processValue = value.values[0];
+    ensureType(processValue, "plain");
+    options.incAfter = {
+      process: processValue.value,
+      token: processValue.token,
+    };
+  } else {
+    ensureType(value, "plain");
+    options.incAfter = undefined;
```

Comment:
Question: Is `INCAFTER()` a valid standalone option? The syntax diagram [here](https://www.ibm.com/docs/en/pli-for-aix/3.1.0?topic=options-incafter) specifies that it should always have a `PROCESS` value in there.

Reply:
I see, thanks I didn't notice. That was just the first result that popped up when searching for `INCAFTER pli` :D